src/master-playlist-controller.js

```diff
@@ -268,7 +268,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       if (event.data.source === 'main') {
         this.mainSegmentLoader_.handleDecrypted_(event.data);
       } else if (event.data.source === 'audio') {
-        this.audiosegmentloader_.handleDecrypted_(event.data);
+        this.audioSegmentLoader_.handleDecrypted_(event.data);
```

Comment:
Thanks @kitce ! Submitted a PR https://github.com/videojs/videojs-contrib-hls/pull/1046 to fix it.